app/views/core/CreateAccountModal/teacher/SetupAccountPanel.coffee

```diff
@@ -6,6 +6,9 @@ SetupAccountPanel = Vue.extend
     saving: true
     error: ''
   }
+  computed:
+    inEU: ->
+      return me.inEU()
```

Comment:
Ideally would use the `me` in store rather than the global Backbone model, but either works.